xbmc/settings/dialogs/GUIDialogSettingsBase.cpp

```diff
@@ -837,22 +840,23 @@ void CGUIDialogSettingsBase::OnClick(BaseSettingControlPtr pSettingControl)
     pSettingControl->Update();
 }
 
-void CGUIDialogSettingsBase::UpdateSettingControl(const std::string &settingId)
+void CGUIDialogSettingsBase::UpdateSettingControl(const std::string &settingId, bool updateDisplayOnly /* = false */)
```

Comment:
> The `ColumnLimit` in `.clang-format` is set to `100` which defines line length [...]

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#3-formatting


> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types